DOCS/man/options.rst

```diff
@@ -3819,6 +3819,31 @@ Demuxer
     (This value tends to be fuzzy, because many file formats don't store linear
     timestamps.)
 
+``--demuxer-hysteresis-secs=<seconds>``
+    Once the ``--demuxer-max-bytes`` prefetch limit is reached, this value can
```

Comment:
I feel like it's awkward having the prefetch limit in bytes but this in seconds. Any reason why this is in seconds and not bytes?

Reply:
The purpose of this is to keep power usage minimized. A watt is one joule per second -- energy over time. Saying "30 megabytes" or "72 kilobytes" is meaningless, since what actually matters is what those byte values amount to. "53 megabytes is 1 second of this movie; the user wants to limit disk activity to once per five seconds, so multiply this out..." is the reasonable decision.